examples/src/main/java/com/opengamma/strata/examples/finance/CalibrationCheckExample.java

```diff
@@ -202,9 +202,11 @@ private static void performance_calibration_pricing() {
         .build();
 
     // calculate the results
-    CalculationRunner runner = CalculationRunnerFactory.ofSingleThreaded()
-        .createWithMarketDataBuilder(trades, columns, rules, marketDataFactory(), marketDataConfig);
-    return Pair.of(trades, runner.calculateSingleScenario(marketSnapshot));
+    return CalculationRunnerFactory.multiThreaded(factory -> {
```

Comment:
The factory needs to be closable because it's the factory that owns the thread pool. If the runners are closable it leaves us with the same problem with have already - the thread pool in the factory is never shut down.